steps/xcode-test-bundle-builder/0.0.3/step.yml

```diff
@@ -0,0 +1,81 @@
+title: Build xcode test runner for remote testing service
+summary: Build your XCTest or XCUITest runner to upload to remote testing service
+description: Build your XCTest or XCUITest bundle to upload to remote testing service
```

Comment:
Circle CI doesn't have a remote testing service AFAIK - did you mean another service?